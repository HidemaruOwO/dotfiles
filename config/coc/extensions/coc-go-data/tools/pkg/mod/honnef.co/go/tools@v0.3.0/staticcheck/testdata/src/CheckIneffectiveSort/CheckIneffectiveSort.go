package pkg

import "sort"

func fn() {
	var a []string
	var b []float64
	var c sort.StringSlice

	a = sort.StringSlice(a)  // want `sort\.StringSlice is a type.+consider using sort\.Strings instead`
	b = sort.Float64Slice(b) // want `sort\.Float64Slice is a type.+consider using sort\.Float64s instead`
	c = sort.StringSlice(c)
}
