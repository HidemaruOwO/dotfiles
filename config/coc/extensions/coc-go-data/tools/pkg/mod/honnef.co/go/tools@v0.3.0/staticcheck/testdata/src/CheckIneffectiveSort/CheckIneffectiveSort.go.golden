package pkg

import "sort"

func fn() {
	var a []string
	var b []float64
	var c sort.StringSlice

	sort.Strings(a)  // want `sort\.StringSlice is a type.+consider using sort\.Strings instead`
	sort.Float64s(b) // want `sort\.Float64Slice is a type.+consider using sort\.Float64s instead`
	c = sort.StringSlice(c)
}
